site/index.qmd

```diff
@@ -197,9 +199,9 @@ The **purpose-built platform** for model risk management teams to test, document
 :::{#admin}
 :::
 
-<div style="text-align: center;">
-<a class="btn btn-primary" role="button" style="font-variant: small-caps;padding: 12px;color: var(--bs-white);background: #083E44;border-radius: 4px;margin-top: 4px;font-family: 'Inter';font-style: normal;font-weight: 700;font-size: 20px;line-height: 100%;border: 1px solid #083E44;" href="guide/guides.html">all user guides</a>
-</div>
+<center>
```

Comment:
Hard "no" on wrapping our new, simpler button in yet more HTML **44 times** in this PR to make things complicated again ... Let's not use HTML when a simpler solution is readily available, please?

In this case, you can add `.tc` to line 189 and `.tl` to line 195 to achieve exactly the same effect with Tachyons. See faccf9191651f60038efec8d63e90523e05a2bb5 for an example on another column to show you what I mean.

Reply:
For this one this ONLY aligns the buttons — not the top lead-in text or the text for the listings for example. I've already simplified the centering for the tachyons flex-boxes with the `.tc` as much as possible with the exception of when the buttons are part of other text and the other text shouldn't be centered.

Reply:
Ah, I see what you mean here, thank you for the example! I forgot the lead-in text was its own element.

Reply:
All the other instances ARE buttons that are part of a larger body text that isn't their own element but I figured we could wrap them in the tachyons css instead of the HTML.

Reply:
> All the other instances ARE buttons that are part of a larger body text that isn't their own element but I figured we could wrap them in the tachyons css instead of the HTML.

Yes, using Tachyons would work though if we're spending a lot of time centering buttons manually, it might be worth chatting about WHY we're doing this. Authoring should be a simple process that is able to focus on the content and not on individually tweaking the appearance of elements.

Reply:
These are just one or two per page in the releases only, not in the core guides. Just the exception with all the images and examples being bumped up against each other in the flex-boxes, etc.